gojek-commons-amqp/src/main/java/com/gojek/amqp/AmqpConfiguration.java

```diff
@@ -21,29 +26,41 @@
 	private Integer minChannels = 5;
 	
 	private Integer maxIdleChannels = 5;
-	
+
+	private Integer networkRecoveryInterval = 1000;
+
+	private List<String> hosts = Lists.newArrayList();
```

Comment:
Initialisation is superfluous. No need of pulling google dependencies just for this.